docs/examples/gen_velocities.rst

```diff
@@ -0,0 +1,432 @@
+      - Remarks
+
+    * - **Coordinates of the pulsar system**
+      -  
+      -
+
+    * - declination
+      - :math:`\delta`
+      -
+
+    * - right ascension
+      - :math:`\alpha`
+      -
+
+    * - proper motion in declination
+      - :math:`\mu_\delta`
+      -
+
+    * - proper motion in right ascension
+      - :math:`\mu_{\alpha*}`
+      - including the :math:`\cos(\delta)` term
+
+    * - **Orbital elements of the pulsar binary**
+      -  
+      -
+    
+    * - binary period
+      - :math:`P_\mathrm{b}`
+      - 
+
+    * - projected semi-major axis
+      - :math:`a_\mathrm{p} \sin( i_\mathrm{p} )`
+      -
+
+    * - orbital inclination
+      - :math:`i_\mathrm{p}`
+      - between :math:`0^\circ` (face-on) and :math:`90^\circ` (edge-on)
+
+    * - longitude of ascending node
+      - :math:`\mathit{\Omega}_\mathrm{p}`
+      - measured from north through east
+
+    * - time of ascending node
+      - :math:`T_\mathrm{asc,p}`
+      -
+
+    * - **Further parameters**
+      -  
+      -
+
+    * - distance to the pulsar system
+      - :math:`d_\mathrm{p}`
+      -
+
+    * - distance to the screen
+      - :math:`d_\mathrm{s}`
+      - from Earth
+
+    * - position angle of the lens
+      - :math:`\mathit{\Omega}_\mathrm{s}`
+      - measured from north through east to the direction of motion on the sky
+
+    * - velocity of the interstellar medium
+      - :math:`v_\mathrm{ISM}`
+      - projected onto the screen direction
+
+.. plot::
+    :include-source:
+    :context:
+
+    p_b = 5.7410459 * u.day
+    asini_p = 3.3667144 * const.c * u.s
+    i_p = 137.56 * u.deg
+    omega_p = 207. * u.deg
+    t0_p = Time(54501.4671, format='mjd')
+
+    d_p = 156.79 * u.pc
+    d_s = 90.6 * u.pc
+    omega_s = 134.6 * u.deg
+    v_ism = -31.9 * u.km / u.s
+
+The coordinates should be placed directly in a
+:py:class:`~astropy.coordinates.SkyCoord` object, that includes the pulsar
+system's position on the sky, its distance, and its proper motion.
+
+.. plot::
+    :include-source:
+    :context:
+
+    psr_coord = SkyCoord('04 37 15.99744 -47 15 09.7170',
+                         unit=(u.hourangle, u.deg),
+                         distance=d_p,
+                         pm_ra_cosdec=121.4385 * u.mas / u.yr,
+                         pm_dec=-71.4754 * u.mas / u.yr)
+
+Calculate some derived quantities:
+
+.. list-table::
+    :widths: 2 1
+    :header-rows: 1
+
+    * - Parameter
+      - Equation
+
+    * - semi-amplitude of the pulsar's projected velocity
+      - 
+        .. math::
+            
+            K_\mathrm{p} = \frac{ 2 \pi a_\mathrm{p} \sin( i_\mathrm{p} ) }
+                                { P_\mathrm{b} }
+
+    * - fractional distance to the screen (from Earth)
+      - 
+        .. math::
+            
+            s = 1 - \frac{ d_\mathrm{s} }{ d_\mathrm{p} }
+
+    * - effective distance
+      - 
+        .. math::
+        
+            d_\mathrm{eff} = \frac{ d_\mathrm{p} d_\mathrm{s} }
```

Comment:
I'd maybe write it in terms of `d_psr` and `s`, since then the analogy with `v_eff` further down will be clearer.

Reply:
Done.